test/utils.js

```diff
@@ -183,6 +185,21 @@ export const separator = length => {
   console.log(`\n${separator}\n`);
 };
 
+export const makePublic = () => {
+  execFileSync(
+    './node_modules/.bin/babel-node',
+    ['./cron/daily/make-private-updates-public.js'],
+    execOptions,
+    (error, stdout, stderr) => {
+      if (error) {
+        logger.error(stderr);
+        throw error;
+      }
+      logger.verbose(stdout);
+    },
+  );
+};
```

Comment:
I would prefer if we would just do something like `models.Update.makePrivateUpdatesPublic()`

Reply:
@brymut Can you address this comment?

Reply:
Hi, I'm sorry. I don't know how I missed this.